frontend/src/vendor/babel-polyfill.js

```diff
@@ -0,0 +1,4 @@
+get:get})},{32:32,39:39,49:49,7:7,70:70,74:74}],208:[function(t,n,r){var e=t(32);e(e.S,"Reflect",{has:function has(t,n){return n in t}})},{32:32}],209:[function(t,n,r){var e=t(32),i=t(7),o=Object.isExtensible;e(e.S,"Reflect",{isExtensible:function isExtensible(t){return i(t),o?o(t):!0}})},{32:32,7:7}],210:[function(t,n,r){var e=t(32);e(e.S,"Reflect",{ownKeys:t(80)})},{32:32,80:80}],211:[function(t,n,r){var e=t(32),i=t(7),o=Object.preventExtensions;e(e.S,"Reflect",{preventExtensions:function preventExtensions(t){i(t);try{return o&&o(t),!0}catch(n){return!1}}})},{32:32,7:7}],212:[function(t,n,r){var e=t(32),i=t(90);i&&e(e.S,"Reflect",{setPrototypeOf:function setPrototypeOf(t,n){i.check(t,n);try{return i.set(t,n),!0}catch(r){return!1}}})},{32:32,90:90}],213:[function(t,n,r){function set(t,n,r){var c,l,h=arguments.length<4?t:arguments[3],v=i.f(a(t),n);if(!v){if(s(l=o(t)))return set(l,n,r,h);v=f(0)}return u(v,"value")?v.writable!==!1&&s(h)?(c=i.f(h,n)||f(0),c.value=r,e.f(h,n,c),!0):!1:void 0===v.set?!1:(v.set.call(h,r),!0)}var e=t(67),i=t(70),o=t(74),u=t(39),c=t(32),f=t(85),a=t(7),s=t(49);c(c.S,"Reflect",{set:set})},{32:32,39:39,49:49,67:67,7:7,70:70,74:74,85:85}],214:[function(t,n,r){var e=t(38),i=t(43),o=t(67).f,u=t(72).f,c=t(50),f=t(36),a=e.RegExp,s=a,l=a.prototype,h=/a/g,v=/a/g,p=new a(h)!==h;if(t(28)&&(!p||t(34)(function(){return v[t(117)("match")]=!1,a(h)!=h||a(v)==v||"/a/i"!=a(h,"i")}))){a=function RegExp(t,n){var r=this instanceof a,e=c(t),o=
```

Comment:
Ah, I think this may be _./node_modules/babel-polyfill/dist/polyfill.min.js_.

Not sure if it's worth using some Gulp task or npm `postinstall` hook, to copy the file from _./node_modules/_ into _./frontend/src/vendor/*_ and then maybe we .gitignore that directory.